dashboard/app/views/lessons/show.html.haml

```diff
@@ -1,4 +1,5 @@
 %link{href: asset_path('css/lessons.css'), rel: 'stylesheet', type: 'text/css'}
+%script{src: webpack_asset_path("js/#{js_locale}/music_locale.js")}
```

Comment:
Should we add a quick comment here explaining why we need this? (I'd love to work towards removing this dependency altogether)